src/e2o/defaultStartup.js

```diff
@@ -0,0 +1,79 @@
+// Electron does not yet support "import"
+const { app } = require('electron');
+const {e2oApplication} = require("@chartiq/e2o")
+//This handles our auto update process
+const E2OUpdater = require('./src/startup/E2OUpdater');
+const log = require('electron-log');
+const superagent = require('superagent');
+const MANIFEST_URL = require('../../configs/other/server-environment-startup.json')[env.NODE_ENV || "development"].serverConfig
+const FEED_URL = 'http://localhost:3375/installers';
+// https://docs.microsoft.com/en-us/windows/desktop/shell/appids
+// Causes the multiple electron apps/windows to be grouped together under a single windows taskbar icon.
+app.setAppUserModelId('e2o');
+
+let updater = null;
+process.on('uncaughtException', (error) => {
+	log.warn(error);
+	log.error('error', error);
+});
+// The installer will do some things here. If it needs to, it will return true and quit the app.(updates...)
+if (require('./src/startup/setupEvents')(app)) { 
```

Comment:
I think loading module and executing code within `if()` is very ugly and considered a bad practice. Why not load at the top of file, and execute inside the if or before it and check the results?

I would only do lazy module loading when I'm not sure which module I need.